### PR TITLE
added function to get joint angles

### PR DESCRIPTION
**Pull request by brahmgardner**

### CODE REVIEW

File: baxter_collaboration_lib/include/robot_interface/robot_interface.h [hunk 1]
```diff
@@ -571,9 +571,10 @@ class RobotInterface
     std::string  getLimb() { return  _limb; };
     State       getState() { return _state; };
 
-    geometry_msgs::Point        getPos()    { return    _curr_pos; };
-    geometry_msgs::Quaternion   getOri()    { return    _curr_ori; };
-    geometry_msgs::Wrench       getWrench() { return _curr_wrench; };
+    geometry_msgs::Point        getPos()         { return    _curr_pos; };
+    geometry_msgs::Quaternion   getOri()         { return    _curr_ori; };
+    geometry_msgs::Wrench       getWrench()      { return _curr_wrench; };
+    sensor_msgs::JointState     getJointStates() { return _curr_jnts; };
```
Comment (alecive) [inline baxter_collaboration_lib/include/robot_interface/robot_interface.h]: Please align curr_jnts with the other variables

File: baxter_collaboration_lib/include/robot_interface/robot_interface.h [hunk 2]
```diff
@@ -571,9 +571,10 @@ class RobotInterface
     std::string  getLimb() { return  _limb; };
     State       getState() { return _state; };
 
-    geometry_msgs::Point        getPos()    { return    _curr_pos; };
-    geometry_msgs::Quaternion   getOri()    { return    _curr_ori; };
-    geometry_msgs::Wrench       getWrench() { return _curr_wrench; };
+    geometry_msgs::Point        getPos()         { return    _curr_pos; };
+    geometry_msgs::Quaternion   getOri()         { return    _curr_ori; };
+    geometry_msgs::Wrench       getWrench()      { return _curr_wrench; };
```
Comment (alecive) [inline baxter_collaboration_lib/include/robot_interface/robot_interface.h:576]: Why did you have to change these three lines? Did you add some tabs/spaces that are not needed?

**Review by alecive — CHANGES_REQUESTED** [review]

File: baxter_collaboration_lib/include/robot_interface/robot_interface.h (hunk 2 above)
Comment (brahmgardner) [reply baxter_collaboration_lib/include/robot_interface/robot_interface.h:576]: i added the tabs/spaces to keep them aligned with the new _curr_jnts variable

Comment (alecive) [reply baxter_collaboration_lib/include/robot_interface/robot_interface.h:576]: :+1:

**Review by alecive — APPROVED** [review]

**Pull request merged**